man/man8/sofdsnoop.8

```diff
@@ -0,0 +1,62 @@
+.TH SOFDSNOOP 8 "2019-07-29" "USER COMMANDS"
+.SH NAME
+sofdsnoop \- traces FDs passed by sockets
+.SH SYNOPSIS
+usage: sofdsnoop [\-h] [\-T] [\-p PID] [\-t TID] [\-n NAME] [\-d DURATION]
+.SH DESCRIPTION
+Trace file descriptors passed via socket
+.SS "optional arguments:"
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+show this help message and exit
+.TP
+\fB\-T\fR, \fB\-\-timestamp\fR
+include timestamp on output
+.TP
+\fB\-p\fR PID, \fB\-\-pid\fR PID
+trace this PID only
+.TP
+\fB\-t\fR TID, \fB\-\-tid\fR TID
+trace this TID only
+.TP
+\fB\-n\fR NAME, \fB\-\-name\fR NAME
+only print process names containing this name
+.TP
+\fB\-d\fR DURATION, \fB\-\-duration\fR DURATION
+total duration of trace in seconds
+.SH EXAMPLES
+.TP
+Trace file descriptors passes
```

Comment:
Maybe "Trace passed file descriptors"?

Reply:
That doesn't sound like something I wrote. It probably comes from the usage message (I used help2man to draft the man page). I'll check and fix it there too if needed.